starter/script.js

```diff
@@ -1 +1,113 @@
+      // --> Start of 1st Iowa 3rd Level <---
+      if (trumpAttack === 'y') {
+        confirm('Donald took the bait and launched one of his Trump attacks.  Good work, you will do well in DC!');
+        //BUG - users picks no, get the correct message but then also get the 'else' prompt.
+      } else if (trumpAttack === 'n') {
+        confirm('You are weak.  Frank might just have to push you in front of a train next time.');
+      } else confirm('You failed to follow simple instrustions.  Reload the page to start over.');
+      // --> End of the 1st Iowa 3rd Level <--
+    } else if (negativeAdSpend => 50)  {
+      var unionStrike = prompt('I like how you operate.  But Bernie is doing too well in New Hampshire.  Do you want to pay a Union leader $50k to go on strike in Vermont to take his attention off the campaign?  Please input "y" for Yes and "n" for No.')
+      console.log("User selected " + unionStrike + " to staging a Union Strike");
+      // --> Start of 2nd Iowa 3rd Level <---
+      if (unionStrike === 'y') {
+        confirm('The Feds caught you in the act.  Off the Federal prison you go.')
+      } else if (unionStrike === 'n') {
+        confirm('Dodged a bullet.  Hilary also sent a team to do this, her campaign manager is off to Federal prison.')
+      } else confirm('You failed to follow simple instrustions.  Reload the page to start over.');
+      // --> End of 2nd Iowa 3rd Level <---
+    // --> End of Iowa 2nd level <---
+  // --> End of Iowa Level <--
+}
+} else if (userPrimaryPick === ('nh')) {
+  var moneyOrSabotage = prompt('We are polling terribly here and Bernie is crushing it.  Should we double our budget or sabotage Bernies campaign? (Please input "m" to double the budget or "s" to sabotage.)"')
+  console.log(userName + " has chosen " + moneyOrSabotage);
+    // --> Start of NH 2nd level <---
+    if (moneyOrSabotage === 's') {
+      var cokeOrAffair = prompt('Now you are thinking like a winner.  Should we plant coke on Bernies campaign manager or pay a loyal supporter to claim she had an affair with him? (Please input "coke" or "affair")')
+      console.log(userName + " chose " + cokeOrAffair)
+      // --> Start of NH's first 3rd Level <--
+      if (cokeOrAffair === 'coke') {
+        confirm('Busted.  Your intern got arrested and ratted you out.');
+      } else if (cokeOrAffair === 'affair') {
+        confirm('Brilliant.  Secretary Clinton jumped all over the accusation and has badly bruised Sanders in the polls, while you keep distance yourself, avoid all suspicion, and continue to rise.');
+      } else confirm('You failed to follow simple instrustions.  Reload the page to start over.');
+      // --> End of NH's first 3rd Level <--
+    } else if (moneyOrSabotage === 'm') {
+      for (i = 0; i < oldArray.length; i += 1) {
+        newArray.push(oldArray[i] = prompt('What 3 things should we spend the increased budget on? ' +  (3 - i) + ' more.'));
```

Comment:
really small note, to save two characters of text you can just use i++, instead of i+= 1!